### PR TITLE
startingDirectory formatting note

### PR DESCRIPTION
**Pull request by linuxfan1**
Finally this time maybe? I think I got it right.

## PR Checklist
* [ ] Closes #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

**Comment by DHowett-MSFT** [comment]
First off, hold up a bit. You don’t need to make a new pull request for every change to the same line of documentation :) you can update existing pull requests probably more easily! This is just general advice going forward, because there might be comments on this one that need addressing!

**Comment by linuxfan1** [comment]
Sorry, kind of new to the public github thing

**Comment by carlos-zamora** [comment]
> Sorry, kind of new to the public github thing
> Do I just commit to my forked repository to update the pull request?

Don't worry about it :) Just `git commit` any changes you want then `git push` to update the pull request.

File: doc/user-docs/UsingJsonSettings.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Example settings include
     "startingDirectory" : "%USERPROFILE%/wslhome"
     ....
 ```
+Note: To use back slashes in the directory you need to escape the `\` as `\\` so that JSON recognizes it. Alternatively you can use `/`.
```
Comment (DHowett-MSFT) [inline doc/user-docs/UsingJsonSettings.md]:
```suggestion

> 👉 **Note**: To use backslashes in any path field, you'll need to escape them following JSON escaping rules (they should look like `\\`). As an alternative, you can use forward slashes.
```

**Pull request merged**